WalletWasabi.Tests/UnitTests/WabiSabi/Models/MultipartyTransactionTests.cs

```diff
@@ -0,0 +1,327 @@
+using NBitcoin;
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using WalletWasabi.Helpers;
+using WalletWasabi.WabiSabi.Backend.Models;
+using WalletWasabi.WabiSabi.Models;
+using WalletWasabi.WabiSabi.Models.MultipartyTransaction;
+using WalletWasabi.Tests.Helpers;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.WabiSabi.Models
+{
+	public class MultipartyTransactionTests
+	{
+		private static MoneyRange DefaultAllowedAmounts = new (Money.Zero, Money.Coins(1));
+		private static Parameters DefaultParameters = new (FeeRate.Zero, DefaultAllowedAmounts, DefaultAllowedAmounts, Network.Main);
+
+		private static void ThrowsProtocolException(WabiSabiProtocolErrorCode expectedError, Action action) =>
+			Assert.Equal(expectedError, Assert.Throws<WabiSabiProtocolException>(action).ErrorCode);
+
+		[Fact]
+		public void TwoPartyNoFees()
```

Comment:
```suggestion
		public void TwoPartiesNoFees()
```
?

Reply:
applying, I will force push after amending with the other lines so leaving unresolved for now